src/NServiceBus.AcceptanceTests/Routing/NativePublishSubscribe/When_publishing_to_scaled_out_subscribers.cs

```diff
@@ -79,7 +79,7 @@ public class SubscriberB : EndpointConfigurationBuilder
         {
             public SubscriberB()
             {
-                EndpointSetup<DefaultServer>();
+                EndpointSetup<DefaultServer>(publisherMetadata: metadata => metadata.RegisterPublisherFor<MyEvent>(typeof(Publisher)));
```

Comment:
since this tests specifies .Repeat(r => r.For<AllTransportsWithCentralizedPubSubSupport>()) there should be no need to register publishers?